src/Bridge/Doctrine/Orm/CollectionDataProvider.php

```diff
@@ -45,7 +46,7 @@ public function __construct(ManagerRegistry $managerRegistry, iterable $collecti
 
     public function supports(string $resourceClass, string $operationName = null, array $context = []): bool
     {
-        return null !== $this->managerRegistry->getManagerForClass($resourceClass);
+        return $this->managerRegistry->getManagerForClass($resourceClass) instanceof EntityManagerInterface;
```

Comment:
@ad3n Actually we missed changing the phpdoc type below: https://github.com/api-platform/core/pull/3005/files#diff-fe622e94e81bf34426323e3ecc2e053eR59

Haha...

Reply:
hahaha you are right, i just delete the line without adding the new one. Sorry for my mistake

Reply:
I will create new PR